testplan/common/entity/base.py

```diff
@@ -31,11 +32,11 @@ class Environment(object):
     """
 
     def __init__(self, parent=None):
-        self._resources = OrderedDict()
-        self.parent = parent
-        self.start_exceptions = OrderedDict()
-        self.stop_exceptions = OrderedDict()
-        self._logger = None
+        self.__dict__["_resources"] = OrderedDict()
+        self.__dict__["parent"] = parent
+        self.__dict__["start_exceptions"] = OrderedDict()
+        self.__dict__["stop_exceptions"] = OrderedDict()
+        self.__dict__["_logger"] = None
 
```

Comment:
since that there's already so much data members and methods in this class, so I think the absence of these 3 will not 马make substantial change but bring extra trouble